app-web/src/components/Search/SearchSources.js

```diff
@@ -22,45 +27,55 @@ export const TEST_IDS = {
   container: 'search.sources.container',
 };
 
-export const SearchSources = () => {
+export const SearchSources = ({ searchSourcesLoading }) => {
   const { authenticated } = useAuthenticated();
   const iconProps = {};
 
-  if (!authenticated) {
+  if (!authenticated || searchSourcesLoading) {
     iconProps.style = {
       opacity: 0.65,
-      margin: '0 2px',
       cursor: 'initial',
-      padding: '0 4px',
+
       boxSizing: 'content-box',
     };
+  } else {
+    iconProps.style = {};
   }
 
   const scrollOffset = -125;
 
   return (
     <SearchSourcesContainer data-testid={TEST_IDS.container}>
-      {authenticated ? (
+      {authenticated || !searchSourcesLoading ? (
```

Comment:
so if the user is authenticated or if search sources have stoppped loading? Is this the intended effect

Reply:
&&